rpc/src/client/sync.rs

```diff
@@ -39,6 +39,7 @@ pub struct ChannelRx<T>(mpsc::UnboundedReceiver<T>);
 impl<T> ChannelRx<T> {
     /// Wait indefinitely until we receive a value from the channel (or the
     /// channel is closed).
+    #[allow(dead_code)]
```

Comment:
This method is not used if only the `http-client` feature is enabled, but putting it behind a feature flag breaks things.